settings.gradle

```diff
@@ -16,3 +16,4 @@
 include 'ds3-sdk-integration', 'ds3-sdk', 'ds3-sdk-samples', 'ds3-interfaces', 'ds3-metadata', 'ds3-utils'
 rootProject.name = 'ds3-java-sdk'
 
+project(":ds3-sdk").name = "ds3_java_sdk"
```

Comment:
is this an alias? or an overall project name rather than a module?

Reply:
It changes the name in gradle without affecting the actual layout in the filesystem.  This was done to keep the name that gets published to bintray consistent without requiring us to move files around in the project.